osh.spec

```diff
@@ -133,6 +139,9 @@ done)
 # collect static files from Django itself
 PYTHONPATH=. osh/hub/manage.py collectstatic --noinput
 
+# set path to python sitelib in the example httpd config
+sed -i 's|@PYTHON3_SITELIB@|%{python3_sitelib}|' osh/hub/osh-hub-httpd.conf.in
```

Comment:
After the in-place edit, the template file is not a template any more.  It would be cleaner to use a separate file for the non-template configuration file:
```
sed 's|@PYTHON3_SITELIB@|%{python3_sitelib}|' osh/hub/osh-hub-httpd.conf.in > osh/hub/osh-hub-httpd.conf
```

Reply:
You're correct, I was just being a bit lazy today. 😅